census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java

```diff
@@ -37,6 +39,20 @@ public final class ObservabilityCensusConstants {
   static final Aggregation AGGREGATION_WITH_BYTES_HISTOGRAM =
       RpcViewConstants.GRPC_CLIENT_SENT_BYTES_PER_RPC_VIEW.getAggregation();
 
+  public static final MeasureDouble API_LATENCY_PER_CALL =
+      Measure.MeasureDouble.create(
+          "grpc.io/client/api_latency",
+          "Time taken by gRPC to complete an RPC from application's perspective",
+          "ms");
+
+  public static final View GRPC_CLIENT_API_LATENCY_VIEW =
+      View.create(
+          View.Name.create("grpc.io/client/api_latency"),
+          "Time taken by gRPC to complete an RPC from application's perspective",
+          API_LATENCY_PER_CALL,
+          AGGREGATION_WITH_BYTES_HISTOGRAM,
```

Comment:
Why is this a `_WITH_BYTES_HISTOGRAM` ? The measure reported is `API_LATENCY_PER_CALL` and no bytes are reported. Wouldn't this be similar to `RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW` i.e. an aggregation over RPC millis?

Reply:
Also if you had a test (similar to `CensusModulesTest.newTagsPopulateOldViews`) to test this new view you would have noticed the wrong type. Is it possible to add a test for the view?

Reply:
You are right, updated measure to use `AGGREGATION_WITH_MILLIS_HISTOGRAM`. 
And added unit test for  `GRPC_CLIENT_API_LATENCY_VIEW`.